src/frontend-samples/clashdetection-sample/ClashDetectionApp.tsx

```diff
@@ -0,0 +1,75 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
This file is supposed to illustrate the main point about the sample.  Yet, there is nothing in here about clash detection.

Reply:
Yes that is a good point. I moved some functions here but this may change more if we can get the REST API calls to succeed.